clang/test/OpenMP/target_teams_codegen.cpp

```diff
@@ -812,6 +812,7 @@ int bar(int n){
 // CHECK1-NEXT:    [[DOTCAPTURE_EXPR__ADDR:%.*]] = alloca i64, align 8
 // CHECK1-NEXT:    [[DOTCAPTURE_EXPR__ADDR2:%.*]] = alloca i64, align 8
 // CHECK1-NEXT:    [[AA_CASTED:%.*]] = alloca i64, align 8
+// CHECK1-NEXT:    [[DOTCAPTURE_EXPR__CASTED:%.*]] = alloca i64, align 8
```

Comment:
@alexey-bataev I didn't change CodeGen at all but why there are these CodeGen changes?

Reply:
Adding a new expression adds a new capture, which is emitted at the codegen

Reply:
Hmm, but I have not added a new expression yet? I have not updated the test case.

Reply:
I think it is because you used the default capture region, which makes passing the value by reference instead of by value

Reply:
Hmm, but that is also what was used before. I didn't change anything.